sqlite/src/expr.c

```diff
@@ -6282,9 +6282,10 @@ static char* sqlite3ExprDescribe_inner(
     }
     case TK_COLUMN: {
       if( atRuntime ){
-        return sqlite3_mprintf("\"%s\"",  pExpr->u.zToken);
+        assert(pExpr->y.pTab && pExpr->y.pTab->nCol > pExpr->iColumn);
+        return sqlite3_mprintf("\"%s\"",  pExpr->y.pTab->aCol[pExpr->iColumn].zName);
       }else{
-        return sqlite3_mprintf("%s",  pExpr->u.zToken);
+        return sqlite3_mprintf("%s",  pExpr->y.pTab->aCol[pExpr->iColumn].zName);
```

Comment:
I think this should also be %q, e.g.:

sqlite> create table t1 ( "x' -- more stuff here" integer );
sqlite> pragma table_info(t1);
0|x' -- more stuff here|integer|0||0

Column names from the schema can include quotes.

Reply:
This does not work in comdb2:
cdb2sql db1 local "create table ttttt (\"x' ha\" integer)"
[create table ttttt ("x' ha" integer)] failed with rc 240 ERROR at line   2: syntax error: '